app/routes/meetings/components/MeetingEditor.js

```diff
@@ -279,11 +280,14 @@ const MeetingEditor = ({
                 ))}
                 {isEditPage && canDelete && (
                   <ConfirmModalWithParent
-                    title="Slett møte"
+                    title="Slett møtet"
                     message="Er du sikker på at du vil slette møtet?"
                     onConfirm={onDeleteMeeting}
                   >
-                    <Button danger>Slett møte</Button>
+                    <Button danger>
+                      <Icon name="trash" size={19} />
+                      Slett møtet
```

Comment:
Also not vertically aligned with the button at its side. And might benefit from a Cancel-button